### PR TITLE
Rename the productDisplayButtons hook to productDisplayAdditionalInfo

### PR DESCRIPTION
**Pull request by antoin-m**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Rename the productDisplayButtons hook to productDisplayAdditionalInfo |
| Type? | improvement |
| Category? | BO & FO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? |  |
| How to test? |  |

### CODE REVIEW

File: install-dev/data/xml/hook.xml [hunk 1]
```diff
@@ -138,8 +138,8 @@
     <hook id="actionOrderReturn">
       <name>actionOrderReturn</name><title>Returned product</title><description>This hook is displayed when a customer returns a product </description>
     </hook>
-    <hook id="displayProductButtons">
-      <name>displayProductButtons</name><title>Product page actions</title><description>This hook adds new action buttons on the product page</description>
+    <hook id="displayProductExtra">
```
Comment (aleeks) [inline install-dev/data/xml/hook.xml]: Need to 1.7.0.0 sql 

UPDATE `PREFIX_hook` SET `name` = 'displayProductExtra' WHERE `name` = 'displayProductButtons';

!

**Review by aleeks — CHANGES_REQUESTED** [review]

**Review by aleeks — CHANGES_REQUESTED** [review]
Need to report on StarterTheme ?

**Comment by Quetzacoalt91** [comment]
Maybe it would be better to find another hook name.

On the PR #6216, we introduced a new hook  displayProductExtra**Content**, which has almost the same description. 😉

**Comment by antoin-m** [comment]
@Quetzacoalt91 the name is the result of a long brainstorming with @julienbourdeau and @maximebiloe . :stuck_out_tongue: 

Do you have something to propose?

**Comment by Quetzacoalt91** [comment]
I don't know for the moment, you made the brainstorming 😆 .

As far as I understand, you modified the hook to make it generic.
However, I almost had this discussion with both @maximebiloe and @julienbourdeau a month ago, which brought the hook I just talked about. And they are almost the same! The only difference is the place where we display it.

Guys, did you already forget ?

**Comment by maximebiloe** [comment]
To be honest, yeah I had forget the hook name 🐟 
@antoin-m & @Quetzacoalt91 what do you think of `displayProductAdditionalInfos`? It's similar with the hook previously added by @Quetzacoalt91 but a little bit more different.

**Comment by Quetzacoalt91** [comment]
Yeah, `displayProductAdditionalInfos`, is okay! Or we just need something after the Extra word. We could try `displayProductExtraSummary` ...

**Comment by Quetzacoalt91** [comment]
Was the hook `displayProductButtons` existing on PS 1.6 ? I think we should  add an alias displayProductButtons -> displayProductAdditionalInfos

**Comment by antoin-m** [comment]
@Quetzacoalt91 done :alien:

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by maximebiloe** [comment]
Okay for me. Need to be reported on `debug`

**Review by aleeks — APPROVED** [review]

**Comment by aleeks** [comment]
When this PR will be merge, please, change hook on ps_viewedproduct module!

**Comment by maximebiloe** [comment]
Thank you @antoin-m 
We finally merge it!

**Pull request merged**